src/main/scala/org/ergoplatform/api/ScriptApiRoute.scala

```diff
@@ -91,6 +114,74 @@ case class ScriptApiRoute(readersHolder: ActorRef, ergoSettings: ErgoSettings)
     }
   }
 
+
+  case class ExecuteRequest(script: String,
+                            env: Map[String,Any],
+                            ctx: ErgoLikeContext)
+    extends TransactionRequest
+
+
+  implicit final val decodeMap: Decoder[Any] = new Decoder[Any] {
+    final def apply(c: HCursor): Result[Any] = {
+      val x = DataJsonEncoder.decode(c.value)
+      Right(x)
+    }
+  }
```

Comment:
@greenhat please help to replace this decoder with AnyValue decoder

Reply:
@aslesarenko `DataJsonEncoder` is in `master` and not in `core-opt`. I suggest to make `v3.0.4` out of master and rebase `core-opt` on top of it.